kubernetes-services/work-scheduler/app/util/env.ts

```diff
@@ -0,0 +1,100 @@
+import camelCase from 'lodash.camelcase';
```

Comment:
I think this should be worked separately as you mentioned. Along with your changes I think that ticket should look to pull common code into a a library (under the harmony repository) that has its own tests and can be used across the sub-projects, instead of the way the sub-projects get built now, which results in a weird file structure.